packages/textlint-fixer-formatter/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "textlint-fixer-formatter",
+  "version": "1.0.0",
+  "description": "textlint output formatter for fixer",
+  "main": "lib/index.js",
+  "types": "src/index.d.ts",
+  "scripts": {
+    "clean": "rimraf lib/",
+    "prepublish": "npm run build",
+    "build": "cross-env NODE_ENV=production tsc -p .",
+    "test": "mocha test"
+  },
+  "files": [
+    "lib/",
+    "src/"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/textlint/textlint.git"
+  },
+  "keywords": [
+    "plugable",
+    "text",
+    "textlint",
+    "markdown",
+    "lint",
+    "linting",
+    "AST"
+  ],
+  "author": "azu",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/textlint/textlint/issues"
+  },
+  "homepage": "https://github.com/textlint/textlint#readme",
+  "devDependencies": {
+    "@types/mocha": "^2.2.44",
+    "@types/node": "^8.0.57",
+    "cross-env": "^5.1.1",
+    "mocha": "^4.0.1",
+    "rimraf": "^2.6.2",
+    "ts-node": "^4.0.1",
+    "typescript": "^2.6.2"
+  },
+  "dependencies": {
+    "@textlint/kernel": "^1.0.3",
+    "chalk": "^2.3.0",
```

Comment:
Previously, textlint use chalk 1.x.x.
Chalk 2.x.x remove `stripColor` method.

> Removed chalk.stripColor(). Use the strip-ansi package directly instead. 04cae22
> https://github.com/chalk/chalk/releases/tag/v2.0.0

Reply:
Oops. I downgraded it version to 1.1.3.